website/index.html

```diff
@@ -190,18 +190,11 @@
       const obj = await ds.assemble(asm);
       const type = obj.frame ? "B" : "C";
       const text = ds.ObjectToDesyncedString(obj, type);
-      let copied = false;
-      if (navigator.userActivation.isActive) {
-        try {
+      try {
         await navigator.clipboard.writeText(text);
-        copied = true;
         alert("Copied to clipboard");
-        } catch {}
-      } 
-      if (!copied) {
-        txtModel.setValue(text);
-        editor.setModel(txtModel);
-        editor.setSelection(txtModel.getFullModelRange());
+      } catch {
+        prompt("Behavior string:", text);
```

Comment:
How does this look if the string is really long? Is it still copyable?

Reply:
The prompt looks like this:

![image](https://github.com/ribrdb/desynced-tools/assets/65694696/88621ef0-8658-4747-8472-604e36d86aea)

In the screenshot I generated a ~6mb string and displayed it using `prompt("Behavior string":, text)`. The string can be fully copied from the prompt dialog.